modules/ROOT/nav.adoc

```diff
@@ -422,6 +422,7 @@ include::cli:partial$cbcli/nav.adoc[]
    *** xref:rest-api:rest-xdcr-pause-resume.adoc[Pausing and Resuming a Replication]
    *** xref:rest-api:rest-xdcr-delete-replication.adoc[Deleting a Replication]
    *** xref:rest-api:rest-xdcr-delete-ref.adoc[Deleting a Reference]
+   *** xref:rest-api:rest-xdcr-identify-target-cluster.adoc[Identifying the Target Cluster]
```

Comment:
"Identifying the Target Cluster" should be changed to "Listing Incoming Replications".